src/briefcase/integrations/adb.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import os
+
+from briefcase.exceptions import BriefcaseCommandError
+
+
+def get_devices(sdk_path, sub=subprocess):
+    # Add our `adb` to start of the path.
+    env = dict(os.environ)
+    env['PATH'] = str(sdk_path / 'platform-tools') + ':' + env.get('PATH', '')
```

Comment:
Any reason to do this by adding to PATH, rather than invoking adb with the explicit path? (I guess they're both equally valid; just interested in the decision process if there was one)

Reply:
My thinking was to run `adb` with its `argv[0]` staying `adb` not `/path/to/adb`. Obsessing over `argv[0]` might be a little silly. Now that I think about it, it's simplest to run it directly rather than monkey with `PATH`, so I'll switch how it works.